lib/compiler.js

```diff
@@ -15,7 +15,17 @@ module.exports = function compile(test, options) {
 
   test = parseFile(test);
 
+  // The following properties are defined by the `test262-parser` module. They
+  // are removed here because they are inconsistent could therefor cause
+  // confusion.
+  delete test.isATest;
+  delete test.async;
+  delete test.strictMode;
+
+  test.scenario = null;
+
   if (test.attrs.flags.raw) {
+    test.insertionIndex = -1;
```

Comment:
this is way easier to read than the ternary you [removed above](https://github.com/bocoup/test262-stream/pull/10/files#diff-71a1f6779f4a9a1fe69fcec81f85db12L28), but I'm a bit out of context and I'm assuming this should be fine, right?

Reply:
Yup!